src/agent/cobertura/Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "cobertura"
+version = "0.1.0"
+edition = "2021"
+license = "MIT"
+
+[dependencies]
+anyhow = "1.0"
+coverage = { path = "../coverage" }
```

Comment:
Ah, so there's a circular dependency now. Can we break the cycle by removing this dependency on `coverage`, and instead move the impl for `From<SourceCoverage> for CoberturaCoverage` into the `coverage` crate?